conf/messages

```diff
@@ -202,13 +202,13 @@ gmp.revaluation_rate = Revaluation rate
 gmp.bulk.results = Results
 gmp.bulk.results.csv = CSV
 gmp.bulk.results.reference = Reference {0}
-gmp.bulk.failure.antivirus = The selected file contains a virus. Create the file again and <a href="/guaranteed-minimum-pension/new-bulk-calculation" id="bulk-back-link">upload it</a>.
+gmp.bulk.failure.antivirus = The selected file contains a virus. Create the file again and <a href="/guaranteed-minimum-pension/new-bulk-calculation" id="bulk-back-link">Select a file in CSV format and upload it</a>.
 gmp.bulk.failure.missing = You did not select a file to upload. Please go <a href="/guaranteed-minimum-pension/new-bulk-calculation">back</a>, browse for a file and try again.
 gmp.bulk.failure.generic = Something went wrong with uploading your file. <br/>There is probably a problem with the system so <a href="/guaranteed-minimum-pension/new-bulk-calculation">try again in a few moments</a>.
 gmp.bulk.failure.duplicate_upload = <span class="lede">We think you have already uploaded this file.</span><br/><br/>We recommend you wait for the results to be ready and do not duplicate file uploads. If you want to process the same file again, perhaps to give it a different name, <a href="/guaranteed-minimum-pension/upload-csv">resubmit it</a>.
 gmp.bulk.failure.too_large = You should include the details of no more than 25,000 members in each file. Try splitting the calculation requests across 2 or more files and <a href="/guaranteed-minimum-pension/upload-csv">upload them</a> separately.
-gmp.bulk.failure.wrong_type = Either no file was selected OR it is the wrong type. Select a file in CSV format and <a href="/guaranteed-minimum-pension/new-bulk-calculation" id="bulk-back-link">upload it</a>.
-gmp.bulk.failure.rejected = Something went wrong with uploading your file. Please check it for any obvious faults, or try to <a href="/guaranteed-minimum-pension/new-bulk-calculation" id="bulk-back-link">upload it</a> again in a few moments.
+gmp.bulk.failure.wrong_type = Either no file was selected OR it is the wrong type. Select a file in CSV format and <a href="/guaranteed-minimum-pension/new-bulk-calculation" id="bulk-back-link">Select a file in CSV format and upload it</a>.
```

Comment:
These are wrong. Make it "No file was selected. <a>Select a file in CSV format and upload it</a>.